holmes/plugins/runbooks/catalog.json

```diff
@@ -2,8 +2,13 @@
   "catalog": [
     {
       "update_date": "2025-06-17",
-      "description": "Runbook to investigate DNS resolution issue on Kubernetes cluster",
+      "description": "Runbook to investigate DNS resolution issue in Kubernetes clusters",
       "link": "networking/dns_troubleshooting_instructions.md"
+    },
+    {
+      "update_date": "2025-07-08",
+      "description": "Runbook to troubleshoot upgrade issues in Azure Kubernetes Service clusters",
```

Comment:
@julia-yin when do we want Holmes to read and use this runbook? What would the user ask?

Reply:
This runbook is used for troubleshooting failed upgrades / upgrade issues in AKS. If a user's cluster is in a failed state or a recent upgrade operation failed, they could ask something like "why did my AKS cluster upgrade fail?" and get a detailed analysis of the failure and remediation steps.